app/adapters/whats_app_adapter/twilio_outbound/template.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module WhatsAppAdapter
+  class TwilioOutbound
+    class Template < ApplicationJob
+      queue_as :default
+
+      def perform(organization_id:, contributor_id:, content_sid:, message:)
```

Comment:
@mattwr18 You sure that serializing objects to Redis won't cause us any problems here? I refer to `message:`

Reply:
Yes, I am sure because the pattern in the project was to pass records into all jobs before I started to refactor away from that pattern. The reason I understand it is a good practice to pass in the id and look up the record is there is a chance that the record changed or was deleted after the job was scheduled, but before the job was executed. 

I think the reason it works is documented here https://guides.rubyonrails.org/v6.1/active_job_basics.html#globalid

That being said, I refactored this job and just pass in the  content_sid and the message_id, look up the message and derive the organization and contributor from it. I think this is an improvement and also addresses a concern that @soey has brought up with paramater drilling.

See 537f3cc4